src/generic/datavgen.cpp

```diff
@@ -5022,20 +5023,25 @@ void wxDataViewMainWindow::UpdateColumnSizes()
     int lastColX = colswidth - lastCol->GetWidth();
     if ( lastColX < fullWinWidth )
     {
-        int desiredWidth = wxMax(fullWinWidth - lastColX, lastCol->GetMinWidth());
-        if ( !lastCol->WXUpdateWidth(desiredWidth) )
```

Comment:
Is this change of behavior, i.e. not returning if the width wasn't changed by `WXUpdateWidth`, intentional?

Reply:
Yes, it's always going to return true now anyhow because `availableWidth > lastCol->GetMinWidth()` if we get here due to the check just above.